org.lflang/src/lib/core/federated/RTI/rti.c

```diff
@@ -590,7 +594,7 @@ void send_provisional_tag_advance_grant(federate_t* fed, tag_t tag) {
  *
  * @return True if the TAG message is sent and false otherwise.
  */
-bool send_tag_advance_if_appropriate(federate_t* fed) {
+bool send_TAG_if_transitively_safe(federate_t* fed) {
```

Comment:
Not sure that the word "transitive" is necessary here.

Reply:
`transitive` refers to [this](https://github.com/lf-lang/lingua-franca/blob/bde2bf2e74871f9255dd34e256aaa771c61e5eb6/org.lflang/src/lib/core/federated/RTI/rti.c#L479) functionality of these functions.

Reply:
Transitivity is the property used to infer whether or not it is safe to advance. It's an implementation detail which doesn't matter to the user of the function. The user cares about signalling federates to advance once safe. Is there another function that's part of the API that does the same thing but ignores transitivity?

Reply:
Yes, we have `send_tag_advance_grant` and `send_provisional_tag_advance_grant`.